src/component/container/pattern-list.tsx

```diff
@@ -62,7 +63,8 @@ export class PatternListContainer extends React.Component<PatternListContainerPr
 	}
 
 	public render(): JSX.Element {
-		const patternRoot = this.props.store.getStyleguide().getPatternRoot() as PatternFolder;
+		const styleguide = this.props.store.getStyleguide() as Styleguide;
+		const patternRoot = styleguide.getPatternRoot() as PatternFolder;
```

Comment:
The only other option here is that `getPatternRoot` can return undefined but especially then it is not a good idea to typecast to something else :)

Reply:
The pattern root can no longer be undefined. I will this the getter type.

Reply:
Done.